BrainPortal/public/stylesheets/boutiques.css

```diff
@@ -30,3 +33,114 @@
   left: 3px;
   color: #ffffff;
 }
+
+/******************************************************************/
+/* Styling for modal preview popup for listing tasks in Boutiques */
+/******************************************************************/
+
+.preview-modal {
+  display: block;
+  position: fixed;
+  z-index: 1;
+  padding-top: 100px;
```

Comment:
Just a general rule about CSS in CBRAIN, we try never to use the unit "pixel", because it doesn't scale well with high-density displays etc. It's better to try to express everything linear distances in, say, 'em's.